packages/next/src/build/index.ts

```diff
@@ -3094,7 +3109,7 @@ export default async function build(
           await fs.rm(exportOptions.outdir, { recursive: true, force: true })
           await fs.writeFile(
             manifestPath,
-            JSON.stringify(pagesManifest, null, 2),
+            formatManifest(pagesManifest),
```

Comment:
I was digging in the manifests enough that having an easy option for me to pretty-print them became a priority. By setting `NEXT_MANIFEST_FORMATTER=pretty` all the manifests will be printed with indentation 😄